network-api/networkapi/wagtailpages/templates/wagtailpages/publication_page.html

```diff
@@ -1,77 +1,137 @@
 {% extends "pages/base.html" %}
-{% load wagtailimages_tags i18n %}
+{% load wagtailcore_tags wagtailimages_tags i18n %}
 
 {% block content%}
-    <div class="container">
-
-        <div
-        class="row mb-5 mt-0 mt-sm-5 d-flex align-items-center">
-            {% comment %} <div class="col-sm-6 mb-3 mb-sm-0">
-                {% image page.hero_image fill-1240x625 %}
-            </div> {% endcomment %}
-
-            <div class="col-sm-10">
-
-                <h1 class="h1-heading">
-                    {{ page.title }}
-                </h1>
-
-                <div class="body-large mb-4">
-                    {% if page.subtitle %}
-                        {{ page.subtitle }}
-                    {% endif %}
-                </div>
-
-                <div class="body-large mb-4">
-
-                    {% if page.secondary_subtitle %}
-                        {{ page.secondary_subtitle }}
-                        {% endif%}
-                    </div>
-                    {% if page.publication_date %}
-                        {{ page.publication_date }}
-                    {% endif %}
-
-
-                    <div class="blog-authors d-flex flex-wrap align-items-center">
-                      <div class="author-images d-flex flex-wrap">
-                      {% for author in page.authors.all %}
-                        <div class="author-image d-flex align-items-center justify-content-center {% if author.image is None %}no-avatar{% endif %}">
-                        {% if author.author.image %}
-                          {% image author.author.image fill-100x100 as img %}
-                          <img src="{{ img.url }}" alt="{{ author.author.name }}" title="{{ author.author.name }}" />
-                        {% endif %}
-                        </div>
-                      {% endfor %}
+<div class="container">
```

Comment:
some debt here: keeping this file formatted properly is painful. it would probably make sense to break it up into some smaller files: 
* hero section template
    * image fragment
    * author fragment
* summary section template

I also think there is some general layout stuff that might be better off in base.html, but I'm not sure since I'm less familar with the codebase overall.